setup.py

```diff
@@ -10,7 +10,20 @@
                     "docopt>=0.6",
                     "Jinja2>=2.7",
                     "PyYAML>=3.10",
-                    "python-dateutil>=2.2"]
+                    "python-dateutil>=2.2",
+                    # IPython dependencies:
```

Comment:
In a project of a library type you shouldn't restrict versioning strictly unless for corner cases.